packages/bp/src/core/migration/migration-service.ts

```diff
@@ -150,9 +149,25 @@ export class MigrationService {
       this.configVersion = yn(process.core_env.TESTMIG_NEW) ? process.BOTPRESS_VERSION : '12.0.0'
     }
 
+    this.dbVersion = await this.getDbVersion()
+
     debug('Migration Check: %o', { config: this.configVersion, db: this.dbVersion, target: this.targetVersion })
   }
 
+  public async getDbVersion(): Promise<string> {
+    let dbVersion: string
+
+    if (process.env.TESTMIG_DB_VERSION) {
+      dbVersion = process.env.TESTMIG_DB_VERSION
```

Comment:
can we directly use `if -> return` pattern there ?

Reply:
I prefer to only have 1 return statement per function